CHANGELOG.md

```diff
@@ -28,6 +28,8 @@
 
 ### Fixes
 
+1. On remote connections check for new or removed LiveKernelConnections on document open.
```

Comment:
Should there be a separate section in the changelog for this point release?

Reply:
Forgot about this. Point releases do usually get a new section.

Reply:
Hmm wonder what the version number should be. Usually we bump the final number for release. It's at 2021.6.99 right now. 2021.6.999?

Reply:
2021.6.ItsOver9000!

Reply:
Personally I don't think it would hurt to omit the version number in the changelog. It's not as though that corresponds to the actual extension version number in the Marketplace.